epic.go

```diff
@@ -79,7 +99,19 @@ type AccountInfo struct {
 }
 
 // Stats is the structure which holds the player's stats for the 3 different game modes offered in Battle Royal.
+type FinalStats struct {
+	Solo			*Stats
```

Comment:
There's some weird formatting going on here. Are you using `gofmt` or `goimports`? It will automatically format your code and fix these issues.

Reply:
Yes sorry I forgot to run it on these. I have more updates coming from my end!